### ISSUE TITLE
Got Error downloading http://127.0.0.1:8000/openapi.json HTTP ERROR 500 after upgrading from 0.14.0 to 0.15.0

### ISSUE THREAD

**Issue by Arka-cell** [tfranzel/drf-spectacular#360]
**Bug Error**
I am having the following bug

<body>
    <redoc spec-url="/openapi.json"><div class="sc-AxjAm bcMPWx"><h1>Something went wrong...</h1><small> Error downloading http://127.0.0.1:8000/openapi.json 
HTTP ERROR 500 </small><p><details><summary>Stack trace</summary><pre>Error: Error downloading http://127.0.0.1:8000/openapi.json 
HTTP ERROR 500
    at https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:29:28051
    at https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:88:36080

Error: HTTP ERROR 500
    at https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:29:28051
    at https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:88:35658</pre></details></p><small> ReDoc Version: 2.0.0-rc.50</small> <br><small> Commit: 8bee3d5</small></div></redoc>

  
<div id="extwaiokist" style="display:none" v="mlgef" q="2cc12d65" c="69.77" i="76" u="134.6" s="03302107" d="1" w="false" e="" m="BMe=" vn="9spdf"><div id="extwaigglbit" style="display:none" v="mlgef" q="2cc12d65" c="69.77" i="76" u="134.6" s="03302107" d="1" w="false" e="" m="BMe="></div></div></body>

The bug happened only when I upgraded from 0.14.0 to 0.15.0. Switching back to 0.14.0 would make everything work as expected.

```python
    path("openapi.json", SpectacularJSONAPIView.as_view(), name="openapi"),
    path("", SpectacularRedocView.as_view(url_name="openapi"), name="redoc"),
```

I also tried to generate schema.yml and use it instead of openapi.json. But I got the following [issue](https://github.com/tfranzel/drf-spectacular/issues/358).

Finally, on my terminal the following error occurs and I think it might be related to the issue previously mentionned. Here is error:

```bash
Internal Server Error: /openapi.json
Traceback (most recent call last):
  File "...\venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "...8\venv\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "...\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "...\venv\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "...\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "...\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "...\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "...\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "...\venv\lib\site-packages\drf_spectacular\views.py", line 67, in get
    return self._get_schema_response(request)
  File "...\venv\lib\site-packages\drf_spectacular\views.py", line 71, in _get_schema_response
    return Response(generator.get_schema(request=request, public=self.serve_public))
  File "...\venv\lib\site-packages\drf_spectacular\generators.py", line 242, in get_schema
    paths=self.parse(request, public),
  File "...\venv\lib\site-packages\drf_spectacular\generators.py", line 216, in parse
    operation = view.schema.get_operation(
  File "...\venv\lib\site-packages\drf_spectacular\openapi.py", line 57, in get_operation
    operation['operationId'] = self.get_operation_id()
  File "...\venv\lib\site-packages\drf_spectacular\openapi.py", line 281, in get_operation_id
    tokenized_path = self._tokenize_path()
  File "...\venv\lib\site-packages\drf_spectacular\openapi.py", line 304, in _tokenize_path
    path = re.sub(
  File "...\AppData\Local\Programs\Python\Python38\lib\re.py", line 208, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "...\AppData\Local\Programs\Python\Python38\lib\re.py", line 302, in _compile
    p = sre_compile.compile(pattern, flags)
  File "...\AppData\Local\Programs\Python\Python38\lib\sre_compile.py", line 764, in compile
    p = sre_parse.parse(p, flags)
  File "...\AppData\Local\Programs\Python\Python38\lib\sre_parse.py", line 948, in parse
    p = _parse_sub(source, state, flags & SRE_FLAG_VERBOSE, 0)
  File "...\AppData\Local\Programs\Python\Python38\lib\sre_parse.py", line 443, in _parse_sub
    itemsappend(_parse(source, state, verbose, nested + 1,
  File "...\AppData\Local\Programs\Python\Python38\lib\sre_parse.py", line 511, in _parse
    sourceget()
  File "...\AppData\Local\Programs\Python\Python38\lib\sre_parse.py", line 256, in get
    self.__next()
  File "...\AppData\Local\Programs\Python\Python38\lib\sre_parse.py", line 245, in __next
    raise error("bad escape (end of pattern)",
re.error: bad escape (end of pattern) at position 1
[08/Apr/2021 10:31:18] "GET /openapi.json HTTP/1.1" 500 156149
```

**Comment by tfranzel** [comment]
hi, there is a bugfix attached to #358 and i just released `0.15.1` with this fix. please update and test again.

**Comment by Arka-cell** [comment]
> hi, there is a bugfix attached to #358 and i just released `0.15.1` with this fix. please update and test again.

It is working, thanks! It was related to python's regular expressions I guess.
